FileShareClient/Models/Attributes.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    public class Attributes : IEquatable<Attributes> 
```

Comment:
Please remove the `IEquatable` implementation from `Attributes` and remove `Equals` , `Equals`, `GetHashCode`
I'm struggling to think of a use case for this and we don't want the maintenance overhead